styles/Vocab/Plone/accept.txt

```diff
@@ -1,10 +1,22 @@
+async
+api 
+backend
+css
+http 
+https 
+JavaScript
+LTS 
```

Comment:
Remove only. Acronyms should not cause Vale to emit warnings. Where do you see Vale emit a warning for `LTS`?

```suggestion
```

Reply:
OK, let's put LTS in here. I want to make sure that when we add configuration, that we do it for a legitimate reason. Also before adding terms wholesale, we should verify that acronyms follow the pattern recommended by Vale to avoid warnings in the first place. See https://vale.sh/docs/topics/styles/#conditional. In this case, LTS is well known, so I won't fuss over it.